test/LexerTest.php

```diff
@@ -39,126 +39,176 @@ public function testLexer($rql, $expected)
     public function lexerProvider()
     {
         return array(
-            'complex eq search concatenated fields' => array('eq(name-part+test,12)', array(
```

Comment:
Please revert you syntax cleanup on this file. I'll be glad to merge the cleanup in another PR. Having it here makes it impossible to review this PR.